### DISCUSSION TITLE
el-slider manual input number

### DISCUSSION THREAD

**Discussion by alvinindra** [element-plus/element-plus#5847] [Q&A] [answered]
I have a problem when manually input number outside el-slider. 

![image](https://user-images.githubusercontent.com/30944414/152956033-b6f6c882-2acc-4510-82e2-76053c36fe32.png)
The bar not updated with current input number. In Vue 2 I can solve this with add @change in el-slider and set manual values in method

![image](https://user-images.githubusercontent.com/30944414/152956373-416e9b34-6c88-4711-a65e-fc77692afda7.png)

Is there any method to solve this in Vue 3?

**Comment by msidolphin** [comment]
This is a bug, we will fix it as soon as possible

**Comment by msidolphin** [comment, marked as answer]
This issue has been fixed https://github.com/element-plus/element-plus/pull/5876
